js/control.js

```diff
@@ -0,0 +1,111 @@
+function loadQuestion(Question){                                                                                   //takes one argument to figure out which questions to load, then displays the next one in order
+  $('#questions').text(Question.questions[questionCount]);
+};
+
+$('.buttons').on('click', '#bartender-game', function(){                                                           //listener for bartender button
+  $('.which-game').hide();                                                                                         //hides the game buttons since game is already decided
+  loadQuestion(questionsToAskDrink);                                                                               //loads the first question, and passes the drink game as argument
+  $('.answer').show();                                                                                             //shows the yes and no answer buttons
+  gamePlayed = {                                                                                                   //fills the gamePlayed object with the questions and pantry to load
+    questions : questionsToAskDrink,
+    pantry : pantryDrink
+  };
+});
+
+$('.buttons').on('click', '#burger-game', function(){
+  $('.which-game').hide();
+  loadQuestion(questionsToAskBurger);
+  $('.answer').show();
+  gamePlayed = {
+    questions : questionsToAskBurger,
+    pantry : pantryBurger
+  };
+});
+
+
+$('.buttons').on('click', '.answer', function(){                                                                    //listener for the yes and no answer buttons
+  var yesOrNo = $(this).text();                                                                                     //gets the value of button pressed
+  switch(questionCount){                                                                                            //checks to see how many questions have been answered, increments the counter each time, and loads the next question
+    case 0:
+      userPreferences.first = yesOrNo;
+      questionCount++;
+      loadQuestion(gamePlayed.questions);
+      break;
+    case 1:
+      userPreferences.second = yesOrNo;
+      questionCount++;
+      loadQuestion(gamePlayed.questions);
+      break;
+    case 2:
+      userPreferences.third = yesOrNo;
+      questionCount++;
+      loadQuestion(gamePlayed.questions);
+      break;
+    case 3:
+      userPreferences.fourth = yesOrNo;
+      questionCount++;
+      loadQuestion(gamePlayed.questions);
+      break;
+    case 4:
+      userPreferences.fifth = yesOrNo;
+      questionCount++;
+      hideEverything();                                                                                             //once all five are answered it hides everything, passes to the mixItem method
+      mixer.mixItem();
+      break;
+    default:
+      console.log('error with switch and counter');
```

Comment:
Just curious about this: I'm guessing the default "case" is to log an error in the console if something goes awry? This seems like a good idea, but I've also been told to remove debugging code?